Remotion/Web/Development.WebTesting.UnitTests/Configuration/WebTestConfigurationSectionTest.cs

```diff
@@ -0,0 +1,143 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Configuration;
+using System.IO;
+using System.Xml.Schema;
+using NUnit.Framework;
+using Remotion.Development.UnitTesting.Configuration;
+using Remotion.Web.Development.WebTesting.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests.Configuration
+{
+  [TestFixture]
+  public class WebTestConfigurationSectionTest
+  {
+    private const string c_fullConfigurationXml = @"
+<remotion.webTesting xmlns=""http://www.re-motion.org/WebTesting/Configuration/1.0""
+    browser=""Chrome""
+    searchTimeout=""00:00:43""
+    retryInterval=""00:00:00.042""
+    webApplicationRoot=""http://some.url:1337/""
+    screenshotDirectory="".\SomeScreenshotDirectory""
+    downloadStartedTimeout=""00:00:13""
+    downloadUpdatedTimeout=""00:00:37""
+    logsDirectory="".\SomeLogsDirectory""
+    closeBrowserWindowsOnSetUpAndTearDown=""false""
+    cleanUpUnmatchedDownloadedFiles=""false""
+    requestErrorDetectionStrategy=""requestErrorDetectionStrategy"">
+  <hosting name=""IisExpress"" type=""IisExpress"" path="".\..\..\..\Development.WebTesting.TestSite"" port=""60042"" />
+</remotion.webTesting>";
+
+    private WebTestConfigurationSection _section;
+
+    public WebTestConfigurationSectionTest ()

```

Comment:
Checked with lead, we don't do empty ctors in test classes

Reply:
As for the reasoning, we'd want to have it if we'd use xUnit, but in NUnit all the important setup stuff happens in `[SetUp]` and everything that isn't a test is noise that should be kept to a minimum. In addition, every test fixture needs a public default ctor to work anyway, so we leave it out

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/14/commits/8b5d5595c2e5fac5e838740df8ca0e105f6e5e3b.